components/layout/fragment.rs

```diff
@@ -882,16 +882,26 @@ impl Fragment {
         let size = LogicalSize::new(self.style.writing_mode,
                                     split.inline_size,
                                     self.border_box.size.block);
-        let flags = match self.specific {
-            SpecificFragmentInfo::ScannedText(ref info) => info.flags,
-            _ => ScannedTextFlags::empty()
+        let (flags, insertion_point) = match self.specific {
```

Comment:
Could you add a comment here explaining what this code does with the insertion point?